lib/presentation/mixins/contacts_view_controller_mixin.dart

```diff
@@ -21,42 +28,83 @@ mixin class ContactsViewControllerMixin {
     const Right(ContactsInitial()),
   );
 
-  final warningBannerNotifier = ValueNotifier<WarningContactsBannerState>(
-    WarningContactsBannerState.hide,
+  final presentationPhonebookContactNotifier =
+      ValueNotifier<Either<Failure, Success>>(
+    const Right(GetPhonebookContactsInitial()),
   );
 
+  ValueNotifier<WarningContactsBannerState> get warningBannerNotifier =>
+      contactsManager.warningBannerNotifier;
+
   final FocusNode searchFocusNode = FocusNode();
 
-  final Debouncer _debouncer = Debouncer(
+  final Debouncer<String> _debouncer = Debouncer(
```

Comment:
```suggestion
  final Debouncer<String> _filterContactDebouncer = Debouncer(
```

Reply:
no, it is text editing debouncer

Reply:
But you use it to filter the contact list